### PR TITLE
added_assigned_to_type_attribute

### PR DESCRIPTION
**Pull request by AmirPracti**
Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes/_instances.md [hunk 1]
```diff
@@ -149,10 +151,13 @@ data/attributes/test-id* | Test id (not display id) | true |
 data/attributes/planned-execution | date field of planned-execution | false |
 data/attributes/version | string of instance version | false |
 data/attributes/priority | string of instance priority | false |
-data/attributes/assigned-to-id | user assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-id | user or group assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-type | assigned-to type (user or group) | false |
```
Comment (yaniviny) [inline source/includes/_instances.md:155]: Should show that thee default is User

File: source/includes/_instances.md [hunk 2]
```diff
@@ -267,9 +273,12 @@ Available parameters | Description |
 data/attributes/planned-execution | date field of planned-execution |
 data/attributes/version | string of instance version |
 data/attributes/priority | string of instance priority |
-data/attributes/assigned-to-id | user assigned-to id (not Display ID) - [users list](#users)  |
+data/attributes/assigned-to-id | user or group assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-type | assigned-to type (user or group) | false |
```
Comment (yaniviny) [inline source/includes/_instances.md:277]: here as well.

File: source/includes/_sets.md [hunk 3]
```diff
@@ -232,13 +237,18 @@ This endpoint updates a specific TestSet.
...
+data/attributes/assigned-to-id | user or group assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-type | assigned-to type (user or group) | false |
 data/attributes/planned-execution | date field of planned-execution |
 data/attributes/version | string of TestSet version |
 data/attributes/priority | string of TestSet priority |
 data/attributes/custom-fields | a hash of custom-fields with their value |
 data/attributes/tags | an array of tags | false |
 
+* assigned-to-id and assigned-to-type are not required. If only assigned-to-id is passed, then type will be considered as user.
+So when assign to a group is wanted, assigned-to-type group must be passed.
```
Comment (yaniviny) [inline source/includes/_sets.md:249]: this is re-written in duplicated places.
maybe it should be placed somewhere on top, talking about assigned-to field, and how to approach it?
and then link to [see more here]

**Review by yaniviny — CHANGES_REQUESTED** [review]
I would say that now we actually need for groups api (like we have users).

File: source/includes/_instances.md (hunk 1 above)
Comment (AmirPracti) [reply source/includes/_instances.md:155]: is this sentence (from the bullet under the parameters table) not enough?
>> If only assigned-to-id is passed, then type will be considered as user.

File: source/includes/_sets.md (hunk 3 above)
Comment (AmirPracti) [reply source/includes/_sets.md:249]: do you mean like for the > API rate limit

File: source/includes/_instances.md (hunk 2 above)
Comment (AmirPracti) [reply source/includes/_instances.md:276]: done

File: source/includes/_sets.md (hunk 3 above)
Comment (AmirPracti) [reply source/includes/_sets.md]: done

**Pull request merged**